packages/fulfillment/src/models/fullfilment-set.ts

```diff
@@ -25,6 +40,20 @@ export default class FulfillmentSet {
   @Property({ columnType: "text" })
   name: string
 
+  @Property({ columnType: "text" })
+  type: string
```

Comment:
question: Do we want this to be fully open-ended?

Reply:
for now it is not decided but we can come up with a set of applicable values

Reply:
👍